runelite-client/src/main/java/net/runelite/client/ui/overlay/OverlayPosition.java

```diff
@@ -45,5 +45,13 @@
 	/**
 	 * Place overlay in the bottom right most area possible
 	 */
-	BOTTOM_RIGHT;
+	BOTTOM_RIGHT,
+	/**
+	 * Place overlay directly above left most area of chatbox possible
+	 */
+	ABOVE_CHATBOX_LEFT,
```

Comment:
ABOVE_CHATBOX_LEFT isn't needed because the position is exactly same as the BOTTOM_LEFT